.github/workflows/validate_datasets.yml

```diff
@@ -60,6 +60,23 @@ jobs:
         bash -c "npm install -g bids-validator-*.tgz"
         popd
 
+    - uses: denoland/setup-deno@v1.1.2
+      if: "matrix.bids-validator == 'master-deno'"
+      with:
+        deno-version: v1.x
+
+    - name: Install BIDS validator (master deno build)
+      if: "matrix.bids-validator == 'master-deno'"
+      run: |
+        pushd ..
+        git clone --depth 1 https://github.com/bids-standard/bids-validator
+        echo -e '#!/bin/sh\n'"$PWD/bids-validator/bids-validator/bids-validator-deno \"\$@\"" >/usr/local/bin/bids-validator
```

Comment:
on windows doesn't work since it isn't some SYS environment seems to me

```
Out-File: D:\a\_temp\3284ba5a-ae7c-47a9-8526-af7ee263d32d.ps1:4
Line |
   4 |  echo -e '#!/bin/sh\n'"$PWD/bids-validator/bids-validator/bids-validat …
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Could not find a part of the path 'D:\usr\local\bin\bids-validator'.
```

any quick ideas @effigies on how to make such thing more portable? i.e. place such bids-validator wrapper into the PATH across systems?